torch/nn/functional.py

```diff
@@ -33,8 +33,9 @@
     bias: optional bias of shape :math:`(\text{out\_channels})`. Default: ``None``
     stride: the stride of the convolving kernel. Can be a single number or
       a one-element tuple `(sW,)`. Default: 1
-    padding: implicit zero paddings on both sides of the input. Can be a
+    padding: implicit zero or circular paddings on both sides of the input. Can be a
       single number or a one-element tuple `(padW,)`. Default: 0
+    padding_mode: input paddings accepts values `zeros` or `circular`. Default: `zeros`
```

Comment:
nit: "input paddings accepts"? How about "what kind of padding is applied to both sides. Can be"

Reply:
Oh, I think that is a typo. I meant the type of padding `circular` or `zeros`. I'll fix it. Thank you for catching